src/components/shared/Entity/Actions/Save.tsx

```diff
@@ -96,95 +96,105 @@ function EntityCreateSave({
-                        title = `${messagePrefix}.testNotification.title`;
-                    }
-
-                    showNotification({
-                        description: intl.formatMessage({
-                            id: description,
-                        }),
-                        severity: 'success',
-                        title: intl.formatMessage({
-                            id: title,
-                        }),
-                    });
-
-                    trackEvent(logEvent, payload);
-
-                    await endSubscription(subscription);
+                const formStatus = dryRun
+                    ? FormStatus.TESTED
+                    : FormStatus.SAVED;
+
+                setPubId(payload.id);
+                setFormState({
+                    status: formStatus,
+                    exitWhenLogsClose: !dryRun,
+                });
+
+                let description, title;
+
+                if (!dryRun) {
+                    description = `${messagePrefix}.createNotification.desc`;
+                    title = `${messagePrefix}.createNotification.title`;
+                } else {
+                    description = `${messagePrefix}.testNotification.desc`;
+                    title = `${messagePrefix}.testNotification.title`;
                 }
+
+                showNotification({
+                    description: intl.formatMessage({
+                        id: description,
+                    }),
+                    severity: 'success',
+                    title: intl.formatMessage({
+                        id: title,
+                    }),
+                });
+
+                trackEvent(logEvent, payload);
             },
             async (payload: any) => {
-                if (payload.logs_token === logTokenVal) {
-                    trackEvent(logEvent, payload);
+                trackEvent(logEvent, payload);
 
-                    onFailure({
-                        error: {
-                            title: `${messagePrefix}.save.failedErrorTitle`,
-                        },
-                    });
-
-                    await endSubscription(subscription);
-                }
-            },
-            true
+                onFailure({
+                    error: {
+                        title: `${messagePrefix}.save.failedErrorTitle`,
+                    },
+                });
+            }
         );
-
-        return subscription;
     };
 
     const save = async (event: React.MouseEvent<HTMLElement>) => {
         event.preventDefault();
 
         resetFormState(status);
 
-        const response = await createPublication(
-            draftId,
-            dryRun ?? false,
-            entityDescription
-        );
-        const publicationsSubscription = waitForPublishToFinish(
-            response.data[0].logs_token
-        );
-        if (response.error) {
-            onFailure(
-                {
+        if (draftId) {
```

Comment:
Random wonder if this was ever blank and caused issues. So adding a check for a little while. If we don't see any logging in LogRocket then we can remove this soon.